libr/bin/format/mach0/fatmach0.c

```diff
@@ -48,15 +54,16 @@ struct r_bin_fatmach0_arch_t *r_bin_fatmach0_extract(struct r_bin_fatmach0_obj_t
 		eprintf ("Skipping corrupted sub-bin %d arch %d\n",
 			idx, bin->archs[idx].size);
 		free (buf);
-		ret->b = r_buf_new ();
-		ret->offset = bin->archs[idx].offset;
-		ret->size = bin->archs[idx].size;
-		return ret;
+		//ret->b = r_buf_new ();
+		//ret->offset = bin->archs[idx].offset;
+		//ret->size = bin->archs[idx].size;
+		free (ret);
```

Comment:
@radare check these changes because maybe the fatmach malformed from jailbreak is not loaded anymore